ONSdigital/eq-author-app
# Ear 684 disable skip logic on first question

### What is the context of this PR?

> https://collaborate2.ons.gov.uk/jira/browse/EAR-684
> 
> 

### How to review 

> Add to the list below as appropriate, including screenshots when necessary

* Create a questionnaire with a number of questions and at least two sections
* Check the Add skip logic button is disabled only for first question in the questionnaire
* Create a skip logic on second question
* Move question to become first below and check skip condition has been deleted
* Create skip logic on second question
* Delete first question
* Check the skip has been removed on the new first question
* Create skip logic on first question in second section
* Move second section to become first section
* Check skip logic has been removed from first question of first section
* Create skip logic on first question of second section
* Delete the first section
* Check the skip logic has been deleted from first question in first section

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process
